torch/nn/quantized/__init__.py

```diff
@@ -1,4 +1,2 @@
-
-from __future__ import absolute_import, division, print_function, unicode_literals
-
-from .modules import *  # noqa: F401
+from . import functional  # noqa: F401
```

Comment:
what is this change? revert?

Reply:
Not sure how this got exported. I'll revert